National Gallery of Australia.js

```diff
@@ -0,0 +1,98 @@
+	"creator": "Ben Swift",
+	"target": "^https?://searchthecollection\\.nga\\.gov\\.au/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-08-18 02:43:18"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Ben Swift
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("uniqueId=")) {
+		return "artwork";
+	}
+
+	return false;
+}
+
+function parseArtwork(doc, url){
+
+	var selectorMap = {
+		artist: ".main-artist-name > span:nth-child(1) > a:nth-child(1)",
+		title: ".object-info-big-title",
+		date: ".object-info-big-date > a:nth-child(1)",
+		artworkMedium: ".object-info-left-col > div:nth-child(3) > div:nth-child(2)",
+		artworkSize: ".object-info-left-col > div:nth-child(4) > div:nth-child(2) > div:nth-child(1)",
+		callNumber: ".object-nav-vra-items > div:nth-child(1) > span:nth-child(2)", // Internal Reference Number (IRN)
+		libraryCatalog: ".object-nav-vra-items > div:nth-child(2) > span:nth-child(2)", // Accession number
+	};
+
+	var newArtwork = new Zotero.Item("artwork");
+	var el;
+
+	for (var field in selectorMap) {
+		el = doc.querySelector(selectorMap[field]);
+
+		if (el){
+			newArtwork[field] = el.textContent;
+		} else {
+			Zotero.debug(`Warning: selector for "${field}" didn't match any elements on page.`);
+		}
+	}
+
+	// creators is actually an array...
+	// this only handles the single artist case for now
+	if (newArtwork.artist) {
+		newArtwork.creators.push(ZU.cleanAuthor(newArtwork.artist, "artist"));
+		delete newArtwork.artist;
+	}
+
+	// ad-hoc cleanup operations
+	// if you think *this* is gross, have a look at the markup on the website that it's trying to make sense of...
+	if (newArtwork.artworkMedium) {
+		newArtwork.artworkMedium = newArtwork.artworkMedium.replace(/[\t\n]/g,'').replace(/,/g, ", ");
+	}
+	if (newArtwork.date) {
+		newArtwork.date = newArtwork.date.replace(/[()]/g,'');
+	}
+
+	// finally, construct a URL
+	newArtwork.url = `https://searchthecollection.nga.gov.au/object?uniqueId=${newArtwork.libraryCatalog}`;
+	newArtwork.complete();
+
+	return newArtwork;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "artwork") {
+		parseArtwork(doc, url);
+	}
+}
```

Comment:
Add tests using Scaffold: navigate to a page in the Browser tab, go to the Testing tab, and click New Web and then Save.